_posts/2020-02-03-ftp.md

```diff
@@ -0,0 +1,80 @@
+
+Since the development of FTP and TCP in the 1970s, cybersecurity has become a growing concern. FTP does not encrypt data and by default, neither does TCP. This means that even if you use a username and password over FTP, your data can still be compromised. Here we'll underline why exactly FTP is insecure and alternatives that you can use to keep your data, especially your credentials, secure.
+
+**Reading time: 5 min**
+
+## What is FTP and why is it insecure?
+
+FTP is a system that allows for the transfer of data and files between two systems, a server and a client. The server acts as a data store that the client can access and exchange files between and usually uses port 21. It was developed in the 1970s by Abhay Buhshan as an extension to the TCP/IP protocol in order to transfer files. At the time, it was revolutionary, allowing a user to remotely access data. However, this was back when computers were simpler and text-based and it didn't take into account security, at least not from a hacker's perspective. 
+
+FTP now is a fairly redundant mode of file transfer. It is inefficient, and although it has been updated somewhat over the years, it is still an old protocol designed for non-graphical systems. The problem lies in the fact that many applications still use FTP today, despite support being phased out by various distributors including the [Debian Project](https://www.debian.org/News/2017/20170425). 
+
+But why is it insecure? On the surface, it looks as though FTP can be secured through username and password. However, this isn't really the case, or rather it's not strictly speaking secure. This is in part because FTP doesn't actually *encrypt* its data. Any authentication data is transferred as a plaintext file making it particularly vulnerable to spoofing. In other words, an attacker could 'listen' to your packets, gaining information regarding your login credentials. This is because of the plaintext format, your username and password can be easily read and used to breach your system. Beyond this as well is the fact that FTP is no longer maintained, it has not actually been updated to be compatible with the modern world.
+
+This begs the question: why is FTP still used? Primarily because it's good for large quantities of data and is easily automated. but mostly because it is free and open source. In instances where security isn't a priority, FTP can be good enough, but unfortunately it is still being used in instances where data protection should be of [utmost importance](https://www.healthcareitnews.com/news/205000-patient-records-exposed-misconfigured-ftp-server). 
```

Comment:
The problem is largely around security. FTP *can* be secure, if using SFTP, but it is more complicated to configure for some users.